src/riverroute/mosart_histflds.F90

```diff
@@ -139,8 +139,8 @@ subroutine mosart_histflds_init(begr, endr, ntracers)
       end do
 
       call mosart_hist_addfld (fname='STORAGE_MCH', units='m3',  &
-           avgflag='A', long_name='MOSART main channelstorage', &
-           ptr_rof=h_volr_mch, default='inactive')
+           avgflag='I', long_name='MOSART main channelstorage', &
```

Comment:
Similar to RTM, why does this change from A to I? 

And again you probably want this to remain default inactive.

Reply:
Unlike the CLM, I found that MOSART and RTM do not have the `history_tape_in_use` capability. By default MOSART and RTM write only non-instantaneous fields to history, so both models throw an error about h0i being empty. I decided:
- not to refactor MOSART and RTM (as out of scope and because I think we consider them dead-end models) but
- instead to add one instantaneous field per model so that h0i will not be empty

The alternative would be to refactor both models to include clm's `history_tape_in_use` capability.

Reply:
Erik agreed with my decision, and I will a comment in mosart and in rtm near the variable that I changed to 'I' explaining the implications.